README.md

```diff
@@ -11,34 +11,90 @@ ChronoLog aims to make this easier by generating a timeline of where people are
+the [Using ChronoLog to process and visualise your own text](#using-chronolog-to-process-and-visualise-your-own-text)
+section.
 
-### Using ChronoLog to process and view your own text
+### Using ChronoLog to process and visualise your own text
 
-TODO(#58)
+If the book you want to visualise is not included in the [Chronolog library](https://www.chronolog.co.uk/library/), you
+can process and visualise the text on your own machine.
+
+The following guide assumes that you are using a Unix-like machine.
+
+#### Processing Text
+
+1. Clone the ChronoLog repository to your own machine by running the following command in the terminal:  
+   `git clone https://github.com/JamesLee-Jones/ChronoLog.git`
+2. Navigate to the outermost folder, nlp-timeline-gen  
+   `cd nlp-timeline-gen`
+3. Save the text you want to process and visualise as a `.txt` file. You can save the text in the `/texts/` folder.  
+   For this guide, we will assume we want to process and analyse the text 'The Great Gatsby', saved
+   at `/texts/the_great_gatsby.txt`.
+4. There is a script in `nlp-timeline-gen` called `chronology.py`.  
+   Run `python ./chronolog.py --help` to see detailed information about the script's usage.  
+   To briefly summarise, the script takes 1 compulsory argument, `filename` (the path to the text you want to analyse)
+   and
+   several optional arguments:
+    * `--chapterRegex CR`, which specifies how your book starts a new chapter. There are several preset chapter
+      formats: `chapter_numeral`, for chapters that look like 'CHAPTER II', `chapter_digit` for chapters that look
+      like "CHAPTER 2", `digit` for chapters that look like "2. ", and `numeral` for chapters that look like "II". It is
+      also possible to specify your own custom chapter format, but that is beyond the scope of this tutorial.  
+      For example, each chapter in 'The Great Gatsby' looks like "II", so we would specify `--chapterRegex numeral`.
+      If no `--chapterRegex` argument is given, the program will default to using `--sections`.
+    * `--sections N` splits the text into N roughly equal sections for analysis, in the case that the book is not
+      chaptered or a chapterRegex is not readily available. This defaults to `N=10`.
+    * `--narrator NARRATOR` should be supplied if the text is written in the first person. Supply the full name of the
+      narrator character.
+      For example, 'The Great Gatsby' is written in first person and the narrator is Nick Carraway, so we should
+      specify `--narrator "Nick Carraway"`</ul>
+      There are additional optional arguments for refining the analysis that can be found through the `--help` command.
+5. Run the script using the appropriate arguments for your text. In our example, this
+   is  
+   `python ./chronolog.py --chapterRegex numeral --narrator "Nick Carraway texts/the_great_gatsby.txt`.  
+   Note the expected order: `python ./chronolog.py [optional arguments] [filename]`
```

Comment:
I actually put the filename first when I use the tool and it still works.

Reply:
Thank you! These and the other issues you pointed out should be fixed now :)